.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,61 +1,40 @@
 ---
 name: ✨ Feature request
 about: Suggest an idea for this project
-
 ---
-
-<!---
-Verify first that your issue/request is not already reported on GitHub.
-THIS FORM WILL BE READ BY A MACHINE, COMPLETE ALL SECTIONS AS DESCRIBED.
-Also test if the latest release, and devel branch are affected too.
-ALWAYS add information AFTER (OUTSIDE) these html comments.
-Otherwise it may end up being automatically closed by our bot. -->
+<!--- Verify first that your feature was not already discussed on GitHub -->
+<!--- This form is processed by Ansibot (machine), complete *all* sections as described -->
 
 ##### SUMMARY
-<!--- Explain the problem briefly -->
+<!--- Describe the new feature/improvement briefly below -->
```

Comment:
Maybe ask for a user story/scenario?

Reply:
This is the summary. See the **ADDITIONAL INFORMATION** section.